src/executor/evaluate/error.rs

```diff
@@ -49,6 +49,9 @@ pub enum EvaluateError {
     #[error("unreachable empty aggregate value: {0:?}")]
     UnreachableEmptyAggregateValue(Aggregate),
 
+    #[error("unreachable iterator unwrap")]
+    UnreachableIteratorUnwrap,
```

Comment:
Quick comment, now you can remove this :)

Reply:
I made a mistake. Thank you